tests/e2e/cairo1/test_cairo1_testing_rust.py

```diff
@@ -0,0 +1,46 @@
+import os
+import pytest
+
+from tests.e2e.conftest import CopyFixture, ProtostarFixture
+
+
+def test_cairo1_rust_test(protostar: ProtostarFixture, copy_fixture: CopyFixture):
+    copy_fixture("cairo1_project", "./cairo1_project")
+    copy_fixture("cairo1/test_a.cairo", "./cairo1_project/tests/test_a.cairo")
+    os.chdir("./cairo1_project")
+
+    result = protostar(["test-rust", "tests/test_a.cairo"])
+
+    assert "test_A: Success" in result
+    assert "test_B: Success" in result
+    assert "test_C: Success" in result
+
+
+def test_no_tests_provided(protostar: ProtostarFixture):
+    with pytest.raises(Exception) as ex:
+        protostar(["test-rust"])
+    assert "No tests provided" in str(ex)
+
+
+def test_no_tests_found(protostar: ProtostarFixture):
+    with pytest.raises(Exception) as ex:
+        protostar(["test-rust", "tests"])
+
+    assert "no such test: tests" in str(ex.value)
+
+
+def test_failing_tests(protostar: ProtostarFixture, copy_fixture: CopyFixture):
+    copy_fixture("cairo1_project", "./cairo1_project")
+    copy_fixture(
+        "cairo1/failing_test.cairo", "./cairo1_project/tests/failing_test.cairo"
+    )
+    os.chdir("./cairo1_project")
+
+    # with pytest.raises(Exception) as ex:
```

Comment:
Can we remove this line?

Reply:
yes, I pointed it out in the other PR, here I forgot, sorry

Reply:
[address feeback etc](https://github.com/software-mansion/protostar/pull/1918/commits/0b1ef91db6dd4db747aa336b04f5241da132ee67)